### ISSUE TITLE
Swift 4.2 release

### ISSUE THREAD

**Issue by freak4pc** [RxSwiftCommunity/RxGesture#82]
Hey @jegnux - Seems like the master branch has Swift 4.2 properly fixed, but a release wasn't cut yet :) 

I could do it myself but preferred leaving it to you :] 

Let me know if I can help!

**Comment by jegnux** [comment]
Hi @freak4pc, thanks for the reminder. I have some minor stuff I'd like to add to the force touch gesture recognizer for this v2.
I'll take care of it before the end of the month if you think it's not too late.

**Comment by freak4pc** [comment]
Thanks @jegnux - 
I think that would be tricky as Xcode 10 GM is already out, and today iOS 12 will be officially rolled out to public users.

What do you think about releasing a release that supports Swift 4.2 today or tomorrow, and then by the end of the month just add the force touch recognizer as a v2.1 ?

**Comment by jegnux** [comment]
What's the issue exactly? Doesn't the pod compile if configured with `SWIFT_VERSION` set to 3.3?

**Comment by freak4pc** [comment]
3.3 ? I'm confused :) 

If a developer uses Xcode 10 and Swift 4.2 the project won't build without the latest release (unless the developer points directly to the GitHub repo).

**Comment by jegnux** [comment]
Sorry, I mean `3.0`. You should be able to comnpile a pod with an older version of swift by adding this to podfile:

```ruby
post_install do |installer|
    swift_30_targets = [
    'RxGesture'
    ]
    swift_41_Targets = [
    ]

    installer.pods_project.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = '4.2'
    end

    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
            if swift_33_targets.include? target.name
                config.build_settings['SWIFT_VERSION'] = '3.0'
            elsif swift_41_Targets.include? target.name
                config.build_settings['SWIFT_VERSION'] = '4.1'
            else
                config.build_settings['SWIFT_VERSION'] = '4.2'
            end
        end
    end

end
```

Doesn't this work?

**Comment by freak4pc** [comment]
We are not referring to Swift 3.0 at all at this point. 
Just 4.0 vs 4.2. 

'm not seeing any reason to ask other developer to do this by hand for a specific targets if the `master` branch already deals with both ?

**Comment by jegnux** [comment]
Yeah yeah absolutely, I just wanted to make sure that they are not totally stuck.

**Comment by freak4pc** [comment]
Definitely not stuck, they can simply direct to the repo
```rb
pod 'RxGesture', :git => 'https://github.com/RxSwiftCommunity/RxGesture', branch: 'master'
```

**Comment by jegnux** [comment]
Ok great. I prepare the 2.0.0 for today then

**Comment by freak4pc** [comment]
Appreciate it :) Thanks !

**Comment by freak4pc** [comment]
Will be resolved by #78. Closing.
